cmd/registry/cmd/export/export_test.go

```diff
@@ -1,24 +1,15 @@
-// Copyright 2022 Google LLC. All Rights Reserved.
```

Comment:
Did you mean to delete the copyright?

Reply:
Nope, I caught that just as you did -- it should be back now.